### PR TITLE
Analysis Pipeline

### PR DESCRIPTION
**Pull request by araval**
This simplifies the example notebook with the help "analysis_pipeline.py". We would like to calculate degradation for a large number of systems in our database, so having a pipeline is more efficient. With the help of this script, we now input system metadata and energy or power time-series and we get degradation values along with confidence intervals as output. 

See for example `docs/pipeline example.ipynb`.

### CODE REVIEW

**Comment by mdeceglie** [comment]
@araval Thanks for the PR! We're also working on some code to streamline the workflow. I'll push that and issue a PR by the end of the week. Then let's all plan to compare notes and figure out how to best put all the pieces together in a consistent way.

File: analysis_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+        '''
+        input_filename: csv with power time-series
+        system_metdata: dictionary with the following keys: 
+                        - systemid   (string. unique identifier) 
+                        - latitude
+                        - longitude
+                        - tilt       (degrees)
+                        - azimuth    (degrees)
+                        - pdc        (DC capacity in Watts)
+                        - tempco     (temperature coefficient)
```
Comment (cwhanse) [inline analysis_pipeline.py]: I would suggest `gamma_pmp` as the keyword

Comment (araval) [reply analysis_pipeline.py]: Thank you, done!

**Comment by cdeline** [comment]
This content is being handled by #117 now.

**Pull request closed without merging**